armi/bookkeeping/snapshotInterface.py

```diff
@@ -56,6 +56,11 @@ def interactEveryNode(self, cycle, node):
         if self.cs["dumpSnapshot"] and snapText in self.cs["dumpSnapshot"]:
             self.o.snapshotRequest(cycle, node)
 
+    def interactCoupled(self, iteration):
```

Comment:
No docstring?

Reply:
The `interactBOL` and `interactEveryNode` didn't have one either so I thought I could get away with it. But I'll just add docstring for all of them like a good citizen :)